### ISSUE TITLE
NYC Citi Bike page quality improvement

### ISSUE THREAD

**Issue by kazunori279** [groovenauts/QueryItSmart#39]
- Please use "NYC Citi Bike" for all the descriptions and labels. It's the formal name.
- Replace the title from "NewYouk City Bike" with "NYC Citi Bike Demand Forecast"
- Please use the following description. "Specify the conditions (day of week, month, weather and temperature) for forecasting the demand for NYC Citi Bike with BigQuery".
- Replace "Please choose a factor" with "Please specify the conditions"
- Improve the UI for specifying the conditions
-- setting default values
-- Using better UI component such as slide bars and icons

**Comment by kazunori279** [comment]
Now the UI looks much much better. This is super cool demo.

Is it possible to make the transition between the heat maps for each time smoother?

**Comment by kazunori279** [comment]
Another ask: Can you change the "Searching..." label to "Analyzing..."? Also, we need to change the description text for searching and finished for demand forecasting.

**Comment by nagachika** [comment]
I've changed label and descriptions for demand forecast  in https://github.com/groovenauts/QueryItSmart/pull/68.

```
Analyzing...
644 bike stations/24 hours and 15.6 GBs processed
in 16.9 seconds with Google BigQuery
```

**Comment by kazunori279** [comment]
I tried super reset, but still see the description for image search for both "searching..." and "finished"

**Comment by nagachika** [comment]
I'm sorry, I forgot to pull new image... Now it's deployed to demo site.

**Comment by nakanori** [comment]
>Is it possible to make the transition between the heat maps for each time smoother?

Fixed by #69
